### DISCUSSION TITLE
活动｜隐语成长报告，好礼回馈社区

### DISCUSSION THREAD

**Discussion by halosecret00** [secretflow/secretflow#168] [General]
关于1024程序员日：1024是2的十次方，二进制计数的基本计量单位之一。程序员（英文Programmer）是从事程序开发、维护的专业人员。程序员就像是一个个1024，以最低调、踏实、核心的功能模块搭建起这个科技世界。1GB=1024M，而1GB与1级谐音，也有一级棒的意思。

各位社区小伙伴，1024节日快乐呀！✿✿ヽ(°▽°)ノ✿

隐语SecretFlow于今年7月正式发布，非常开心能得到各界的关注和关爱，三个多月时间里，我们一起在社区交流探索、一起快速成长。在1024这天，让我们来回顾每个瞬间吧：

7月：“隐语SecretFlow”发布会，直播观看**超万人**，互动人数**3000+**！首批代码贡献**from双子座实验室**：**猎豹协议**，目前业界最快的两方安全计算协议！
8月：首次问卷调研：**68**份有效信息，**91**%的支持与鼓励
9月：隐语开放平台上线，至今已有**80**+申请机构！
10月：隐语Github主页：issue数突破**100**+，star数即将迎来**1024**星！✨

期间，隐语社区认真倾听小伙伴们的声音：

——开发者对代码的认可，是我们更加坚定和努力的动力来源！
——对于社区小伙伴问题反馈都尽可能快速应答，用代码和文字让大家感知到社区的温度；一起进步吧
——成长路上，隐语还有诸多不足之处，无论是建议期许还是提交issue都是对隐语社区更深程度的肯定呐~期待大家更多的反馈

所有的问答交流、技术建议、真挚鼓励，这些互动反馈对“隐语”这个刚出生的小“baby”来说，都弥足珍贵

回顾了这些社区的点点滴滴后，是不是越发深刻的感受到隐语社区小伙伴们的热情与真挚了呢？^_^还没结束，因为活动要来喏——在1024这个特殊的日子，为表达社区小伙伴们一直的支持和鼓励，社区准备了惊喜好礼哟！

**抽奖暗号**：**20220704**
**参与方式**：在隐语的小剧场公众号后台或私信隐语小助手回复以上暗号，即可获取抽奖链接，召唤当日份幸运！
![image](https://user-images.githubusercontent.com/109133653/197383409-cb9b92ba-c2c0-4590-9cfb-fdd6861e9fd9.png)

我们为大家准备了丰厚奖品：
一等奖×1：罗技机械键盘（经过隐语社区调研后呼声最高的🎁😄）
二等奖×2：程序员背包（超级实用）
三等奖×3：不锈钢笔记本支架（爱护身体从爱护颈椎开始👦）
感恩奖×5：《隐私保护机器学习》（学习永不停歇📖）

活动时间：10月24日-10月27日，欢迎大家多多交流：https://github.com/secretflow

同时也预告下，隐语社区将在11月份发布【隐私计算线上慕课】系列课程，本系列课程特邀20+名校讲师/嘉宾联合出品，共计20门课程，包含100+细分知识点，届时将在B站陆续上线，欢迎扫码关注我们的bilibili账号：隐语SecretFlow！

![image](https://user-images.githubusercontent.com/109133653/197397760-c033ad01-d18b-43a2-aa89-a6bfa4bd4bc7.png)

**Comment by yeshuchen323** [comment]
已参加~
